acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/tls/support/client/BasicClientTlsOptions.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests.tls.support.client;
+
+import tech.pegasys.ethsigner.core.config.tls.client.ClientTlsCertificateOptions;
+import tech.pegasys.ethsigner.core.config.tls.client.ClientTlsOptions;
+import tech.pegasys.ethsigner.core.config.tls.client.ClientTlsTrustOptions;
+
+import java.util.Optional;
+
+public class BasicClientTlsOptions implements ClientTlsOptions {
+  private final boolean isTlsEnabled;
+  private final Optional<ClientTlsCertificateOptions> clientTlsCertificateOptions;
```

Comment:
nit: no need for the term "client" in the variable name, it's inferred from the class name.

Reply:
I've took the `client` out of variable/method names.